libs/core/src/lib/carousel/carousel.service.ts

```diff
@@ -99,6 +95,9 @@ export class CarouselService implements OnDestroy {
     ngOnDestroy(): void {
         this._onDestroy$.next();
         this._onDestroy$.complete();
+
+        this.activeChange.complete();
+        this.dragStateChange.complete();
```

Comment:
I see, these are not new properties, but still I'd recommend to add `$` sign as a suffix